### PR TITLE
Enable examples/python/simple_local/run.py

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+opencv-python
```
Comment (AlexKoff88) [inline requirements.txt]: Do we really need it on the top level? I changed the one that we have inside "model_api/python/"

File: examples/python/simple_local/run.py [hunk 2]
```diff
@@ -16,19 +16,23 @@
...
     DetectionModel,
     SegmentationModel,
 )
 
 from PIL import Image
 
-sys.path.insert(0, str(Path(__file__).resolve().parents[1] / "tools/model_tools/src"))
-sys.path.insert(0, str(Path(__file__).resolve().parents[1] / "demos/common/python"))
+
+def from_local_xml(image):
```
Comment (AlexKoff88) [inline examples/python/simple_local/run.py:25]: Can you please resolve a conflict? Sorry about that.

File: requirements.txt (hunk 1 above)
Comment (Wovchena) [reply requirements.txt]: Removed

File: examples/python/simple_local/run.py (hunk 2 above)
Comment (Wovchena) [reply examples/python/simple_local/run.py:25]: Done. I also replaced a link to omz_tools to match https://github.com/openvinotoolkit/open_model_zoo/pull/3700 which we are going to merge

**Pull request merged**